manifests/0000_70_dns-operator_04-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: dns-operator
+  name: metrics
+  namespace: openshift-dns-operator
+spec:
+  ports:
+  - name: http
```

Comment:
Should this be secured with cluster cert? (see ingress-operator)

Reply:
I don't think we can secure the metrics endpoint without changes to controller-runtime: https://github.com/kubernetes-sigs/controller-runtime/blob/28262f4974c4c0e0ff421988bb1e44a62bf29b61/pkg/manager/internal.go#L244

Reply:
Good to know, but _should_ we secure it? Are we required to?

Reply:
It is now secured using kube-rbac-proxy.